system_strings/sysmes_text.en

```diff
@@ -6501,47 +6550,71 @@ One week has passed since that^night. Within my unchanging^daily life, I remembe
-……一緒にはいられない。^彼女が吸血鬼だからじゃない。^今の自分には、大切な人がいる。
+// Ref 37614.
+// Copied a bit extra for context.
+// @flowchart_descr
+We can't be together...
+ Not because she's a vampire, but because right now, I have someone I value more than my own life.
 }
 [sha:cb72b0928c8c0839db1d4ecbdd7839fc586f0479]{
 -- アルクェイドが倒れる。^地面が彼女の血で染まっていく。^駆け寄ろうとした、その時――
 Arcueid has fallen. Her blood^stains the ground. I'm about to^run over to her, when―――
 }
 [sha:be5d924a295e911b8d7926f662f0370f132f556d]{
 -- 暴走するアルクェイド。^二人は学校まで撤退し、^戦いに備える。
-暴走するアルクェイド。^二人は学校まで撤退し、^戦いに備える。
+// Second half the same as 38f8a4fe
+// @flowchart_descr
+Arcueid has started to run rampant.
+ Ciel and I retreat to the school, and ready ourselves for battle.
 }
 [sha:0f34a856d20efd289e534ffe2ef37ae5cb6c2bd6]{
 -- 死のない真祖を絶命させる。^それを成すためには、^先に殺すべき物がある……！
-I need to kill the deathless^True Ancestor. But in order to^do that, I must kill something^else first!
+// @flowchart_descr
+I need to kill the deathless True Ancestor.
+ But in order to do that, I must kill something else first!
 }
 [sha:446370fd9a9c3560c24307456d0d4dee392d3435]{
 -- 彼女にナイフを突き立てる。^その傷が自分だけの物であれ、^と願いながら――
-I pierce her with my knife. I^pray that those scars are mine^alone to bear―――
+// @flowchart_descr
+I pierce her with my knife.
+ I pray that those scars are mine alone to bear―――
 }
 [sha:b75f12c5fc4b10f3143bdf364f5969ae08628f5f]{
 -- この命はじきに停止する。^だが、ロアの自由にはさせない。^残された方法はただひとつ。
-My life will soon be at an end.^But I can't allow Roa to roam^free――there is only one thing^left to do.
+// @flowchart_descr
+My life will soon be at an end.
+ But I can't allow Roa to roam free――there is only one thing left to do.
 }
 [sha:f788a3d127b76c2c93e5a3c7779e64f976641378]{
 -- 彼女を嫌いにはなれない。^……だからこそ、^その間違いは見逃せない。
-I can't bring myself to hate^her... But that's also why I^can't overlook this mistake.
+// @flowchart_descr
+I can't bring myself to hate her... But that's also why I can't overlook this mistake.
 }
 [sha:38f8a4fe97826804256cb05ebc3d66a8b8d336ce]{
 -- 本来の力を顕すアルクェイド。^二人は学校まで撤退し、^戦いに備える。
-Arcueid reveals her true power.^The two of us withdraw to the^school, and prepare for battle.
+// Second half the same as be5d924
+// @flowchart_descr
+Arcueid reveals her true power.
+ Ciel and I retreat to the school, and ready ourselves for battle.
 }
 [sha:672cdb4904185359fd8b1f30738b33c7fb68b3ea]{
 -- 第七聖典による狙撃を^敢行するシエル。^渾身の一射は、はたして。
-Ciel uses the Seventh Scripture^to snipe her target. Her first^shot, with all her strength^behind it, goes as expected.
+// @flowchart_descr
+Ciel uses the Seventh Scripture to snipe her target.
+ Her first shot, with all her strength behind it, goes as expected.
 }
 [sha:14bff6ca7cbc7552d9313f4d39e210febb6c17e8]{
 -- 街が熔け、消える。^これは自分と志貴だけの世界。^なのに邪魔者が混ざっている――
-The city melts away and^vanishes. A world for just^Shiki and me. And yet, an^intruder has entered the mix.
+// @flowchart_descr
+The city melts away and vanishes.
+ A world for just Shiki and me.
+ And yet, an intruder has entered the mix.
```

Comment:
ref 39061 & 39062

39062 could probably have 'had' removed just so it works as a continuous sentence better (it'd be okay on its own, but... context)